package.json

```diff
@@ -7,10 +7,14 @@
     "url": "https://github.com/Esya/oasis-wallet.git"
   },
   "scripts": {
-    "start": "react-scripts -r @cypress/instrument-cra start",
+    "start": "yarn start:oasismonitor",
+    "start:oasismonitor": "react-scripts -r ./internals/scripts/set-oasismonitor-backend-env.js -r @cypress/instrument-cra start",
+    "start:oasisscan": "react-scripts -r ./internals/scripts/set-oasisscan-backend-env.js -r @cypress/instrument-cra start",
```

Comment:
I would simplify to
```json
    "start": "react-scripts -r @cypress/instrument-cra start",
    "start:oasismonitor": "REACT_APP_BACKEND=oasismonitor yarn start",
    "start:oasisscan": "REACT_APP_BACKEND=oasisscan yarn start",
```

and `REACT_APP_BACKEND=oasismonitor` in .env, so `yarn start` still works

Reply:
Do we want to keep  "start:oasismonitor": task where it is a duplicate of a default one ?

Reply:
I think so. Default `start` uses REACT_APP_BACKEND from .env, but that can be overwritten by .env.local.

The explicit `start:oasismonitor` ensures REACT_APP_BACKEND is actually oasismonitor